repos/spack_repo/builtin/packages/amd_aocl/package.py

```diff
@@ -25,7 +25,8 @@ class AmdAocl(BundlePackage):
 
     maintainers("amd-toolchain-support")
 
-    version("5.0", preferred=True)
+    version("5.1", preferred=True)
```

Comment:
Preferred doesn't do anything when it's on the most recent version.
```suggestion
    version("5.1")
```

Reply:
Updated as suggested